lib/command.c

```diff
@@ -324,6 +324,7 @@ install_element (enum node_type ntype, struct cmd_element *cmd)
     {
       fprintf (stderr, "Command node %d doesn't exist, please check it\n",
                ntype);
+      fprintf (stderr, "Have you called install_node before this install_element?\n");
```

Comment:
maybe we could even tell the developer what command they installed, with `cmd->string`